ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -415,11 +415,17 @@ def addFiles(self, roleIndex, startingLaneNum=None):
 
     def addFileNames(self, paths: List[Path], startingLaneNum: int, roleIndex: int):
```

Comment:
Assuming that null checks are really needed:

```suggestion
    def addFileNames(self, paths: Optional[List[Path]], startingLaneNum: Optional[int], roleIndex: int):
```